libraries/worker-pool/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
+
+  "compilerOptions": {
+    "target": "ES2019",
```

Comment:
Should we consider updating this in `heft-node-rig`? Are there Node version compatibility considerations?

Reply:
While we were supporting node 10/12 the optional chaining operator was not available, but is part of ES2017. We can probably afford to update, but I'll leave that to its own PR.